source/releases.py

```diff
@@ -0,0 +1,37 @@
+import urllib.request as request, json, os
+from datetime import datetime
+
+def main():
+    response = request.urlopen('https://api.github.com/repos/cyclus/cyclus/releases')
+    releases = json.loads(response.read())
+    table_text = \
+'''.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - Version
+     - Date
+     - Zip
+     - Tar'''
+    for release in releases:
+        version = release['tag_name']
+        date = release['published_at']
+        date_object = datetime.strptime(date, "%Y-%m-%dT%H:%M:%SZ")
+        date_formatted = date_object.strftime("%m/%d/%Y")
+        zip_url = release['zipball_url']
+        tar_url = release['tarball_url']
+        table_text += f'''
+   * - {version}
```

Comment:
How about we add a link to the Github Release page?
```suggestion
        tar_url = release['tarball_url']
        web_url = release['html_url']
        table_text += f'''
   * - `{version} <{web_url}>`
```

Reply:
Good idea, LGTM